lib.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, writeText, runCommand, remarshal }:
+{ lib, writeText, runCommand, remarshal, fetchgit }:
```

Comment:
is `fetchgit` actually used?

Reply:
So I actually got stuck here, since `fetchgit` requires `sha256` and as far as I can tell we don't have that! The `builtins.fetchGit` doesn't, but it's not due for release until Nix 2.4 (Domen told me!)

I could use your guidance or assistance. :)